### DISCUSSION TITLE
Improving helm chart for SigNoz

### DISCUSSION THREAD

**Discussion by prashant-shahi** [SigNoz/signoz#713] [General]
We have migrated helm chart to a new repository: https://github.com/SigNoz/charts

Now, installing SigNoz in your k8s cluster is as easy as running the commands below:
```console
helm repo add signoz https://charts.signoz.io
helm install -n platform --create-namespace my-release signoz/signoz
```
For refer [K8s Installation Docs](https://signoz.io/docs/install/kubernetes/) for more detailed instructions.

Would love any ideas, questions, and feedback on it in this discussion thread.

**Comment by ksponomarev** [comment]
Hello
It will be great if you add customisation for:
-  livenessProbe and readinessProbe for frontend and queryService
- securityContext for all resources

Also there is a problem when I'm trying to drop clickhouse section from `values.yaml`

**Reply by prashant-shahi** [reply]
The above mentioned customisation is included in the current chart.

> Also there is a problem when I'm trying to drop clickhouse section from values.yaml

You would have to set `clickhouse.enabled` to `false`, while you should set `externalClickhouse` section.

**Comment by panovmikhail** [comment]
Hi there.
You can improve this part: https://github.com/SigNoz/charts/blob/1a352acb593af3fd366ce200ec4fd742efc8e1f2/charts/signoz/values.yaml#L1699

**Because, when i added new line i got next situation**

Values.yaml:
```
      include:
        - /var/log/containers/*.log
        - /var/log/pods/*/*/*.log
```
        
Configmap:
```
    include:
    - /var/log/containers/*.log /var/log/pods/*/*/*.log
```

Error:
```
warn fileconsumer/file.go:61 no files match the configured include patterns {"kind": "receiver", "name": "filelog/k8s", "pipeline": "logs", "component": "fileconsumer", "include": ["/var/log/containers/*.log /var/log/pods/*/*/*.log"], "exclude": []}
```

**Reply by prashant-shahi** [reply]
@stok1y Fixed in https://github.com/SigNoz/charts/pull/124
